packages/components/icon/scripts/export-svg.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const { EOL } = require('os');
+const fs = require('fs').promises;
+const { camelize } = require('../../../../scripts/helpers');
+
+/**
+ *  Read all svg files in an assets/icons directory and generate index.js file with ES module exports
+ */
+(async function () {
+	const componentDir = path.join(process.cwd(), '/lib/assets/icons');
+	console.log(componentDir);
+	const files = await fs.readdir(componentDir);
+	let moduleFileContent = '';
+	let kebabCaseExport = 'export const iconList = [' + EOL;
+	for (const file of files) {
+		if (file.endsWith('.svg')) {
+			const iconName = file.split('.')[0];
+			const camelizedName = iconName === 'delete' ? 'deleteIcon' : camelize(iconName);
```

Comment:
Can you please put comment here, to explain what's the reason for doing this for delete icon and if we can get rid of it in first major version release?